docs/email/components/images.html

```diff
@@ -224,5 +230,16 @@
     <p class="stacks-copy">
         Images should be hosted within their respective ESP. Upload images in Iterable for Iterable emails, add images in GitHub for StackMail emails, etc. Images should not be referenced using services like Imgur or Dropbox, as they have a greater chance of being marked as spam.
     </p>
+    {% header h3 | Animated GIFs %}
+    <p class="stacks-copy">
+        Animated GIFs can help explain an interaction or add a bit of delight that isn’t possible with static email designs. However there a couple things to consider when designing and preparing animated GIFs.
```

Comment:
Shouldn't that be _However there **ARE** ..._? My English is meh but this doesn't sound right even to me :DDD